ci/deps/azure-36-minimum_versions.yaml

```diff
@@ -5,26 +5,25 @@ channels:
 dependencies:
   - beautifulsoup4=4.6.0
   - bottleneck=1.2.1
+  - cython>=0.29.13
   - jinja2=2.8
   - numexpr=2.6.2
   - numpy=1.13.3
   - openpyxl=2.4.8
   - pytables=3.4.2
   - python-dateutil=2.6.1
-  - python=3.5.3
+  - python=3.6.0
   - pytz=2017.2
   - scipy=0.19.0
   - xlrd=1.1.0
   - xlsxwriter=0.9.8
   - xlwt=1.2.0
   # universal
   - hypothesis>=3.58.0
+  - pytest=4.5.0
   - pytest-xdist
   - pytest-mock
   - pytest-azurepipelines
   - pip
   - pip:
-    # for python 3.5, pytest>=4.0.2, cython>=0.29.13 is not available in conda
-    - cython>=0.29.13
-    - pytest==4.5.0
     - html5lib==1.0b2
```

Comment:
Can be done as a follow up but not sure why this minimum is pinned to a beta, and I couldn't find anything in our docs for it. Probably safe to move to 1.0.1 which is available on conda